### DISCUSSION TITLE
Is anyone really using this?

### DISCUSSION THREAD

**Discussion by JnCrMx** [JnCrMx/discord-game-sdk4j#10] [Show and tell]
Soooo,

I was wondering if anyone is really using this library for their projects.
And it would be nice to know what people use this for, so I know which features I maybe need to implement in the future and what I can improve.

Therefore, if you are using this library for a project of your own, no matter what kind it is, feel free to share it here :D 
This is also a nice place for ideas about features and improvement or just more general talk and feedback.

**Comment by MrVintage710** [comment]
I was looking at this for a Proximity chat mod mod for Minecraft. The problem is that I would need the Voice manager part of the API, which isn't there yet. I would otherwise!

**Reply by JnCrMx** [reply]
Oki doki, 
then I will definitely work on the Voice manager next.
I'm currently doing Lobbies and Networking, but I'm almost done with it (see #11).

**Reply by RaphyTwin** [reply]
Does this still work? Or did Discord remove that part or the game sdk?

**Comment by hypherionmc** [comment]
I will be using this in my Minecraft Discord RPC mod :)

**Comment by Minecraftian14** [comment]
I am using it in one of my projects called HLTDRP, which itself is not well used but then.... 

Anyway, thanks a lot for maintaining this project : )

**Reply by Minecraftian14** [reply]
So.. my project is placed here:
https://github.com/Minecraftian14/HLTDRP
which is a Discord RPC support plugin for Hyperlap2D at
https://github.com/rednblackgames/HyperLap2D
